File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.cc
#include <iostream>
using namespace std;

int main() {
	int winner;
	int winnersScore = 0;

	int sum = 0;
	for (int i = 0; i < 5; i++)
	{
		int score1;
		int score2;
		int score3;
		int score4;

		cin >> score1 >> score2 >> score3 >> score4;

		sum = score1 + score2 + score3 + score4;

		if (sum > winnersScore)
		{
			winnersScore = sum;
			winner = i + 1;
		}
	}

	cout << winner << " " << winnersScore << endl;
}

File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int winner = 0;
        int winnersScore = 0;

        for (int i = 0; i < 5; i++) {
            int score1 = sc.nextInt();
            int score2 = sc.nextInt();
            int score3 = sc.nextInt();
            int score4 = sc.nextInt();

            int sum = score1 + score2 + score3 + score4;

            if (sum > winnersScore) {
                winnersScore = sum;
                winner = i + 1;
            }
        }

        System.out.println(winner + " " + winnersScore);
    }
}

